src/python/bot/fuzzers/libFuzzer/launcher.py

```diff
@@ -847,10 +765,16 @@ def main(argv):
     if os.path.exists(default_dict_path):
       arguments.append(constants.DICT_FLAG + default_dict_path)
 
+  # Strategy pool is the list of strategies that we attempt to
```

Comment:
I can't tell for sure on github but I think the formatting of these comments is odd. Can you try to fit as many words per line as possible (up to 80 chars)? Thanks!